scss/_accordion.scss

```diff
@@ -0,0 +1,146 @@
+//
+// Base styles
+//
+
+.accordion-button {
+  @include font-size($font-size-base);
+  display: flex;
+  align-items: baseline;
+  width: 100%;
+  padding: $accordion-button-padding-y $accordion-button-padding-x;
+  color: $accordion-button-color;
+  text-align: left;
+  background-color: $accordion-button-bg;
+  border: $accordion-border-width solid $accordion-border-color;
+  border-bottom: none;
+  @include border-radius(0);
+
+  &:not(.collapsed) {
+    color: $accordion-button-active-color !important; // stylelint-disable-line declaration-no-important
+    background-color: $accordion-button-active-bg !important; // stylelint-disable-line declaration-no-important
+    &::after {
+      background-image: escape-svg($accordion-button-active-icon);
+      transform: rotate(180deg);
+    }
+  }
+
+  &::after {
+    flex-shrink: 0;
+    width: 1.5rem;
+    height: 1rem;
+    margin-left: auto;
+    content: "";
+    background-image: escape-svg($accordion-button-icon);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 1.5rem;
+    @include transition($accordion-icon-transition);
+  }
+
+  &:focus,
+  &.focus {
```

Comment:
I don't think the `.focus` class is needed here, you probably copied this from the buttons? They are used there for the button js plugin (not sure if this is even applicable these days, but that 's something else)